vendor/webtreemap.js

```diff
@@ -60,7 +60,7 @@ function focus(tree) {
 function makeDom(tree, level) {
   var dom = document.createElement('div');
   dom.style.zIndex = 1;
-  dom.className = 'webtreemap-node webtreemap-level' + Math.min(level, 4);
+  dom.className = 'webtreemap-node webtreemap-level' + level;
   if (tree.data['$symbol']) {
     dom.className += (' webtreemap-symbol-' +
```

Comment:
I think I would prefer to use $symbol for the color.  source-map-explorer could generate a triple: Source, Size, $symbol.  They in the CLI you can choose what to use to generate $symbol (level, size, name, etc).